rfcs/text/0352-graph-type-system.md

```diff
@@ -18,17 +18,17 @@ Rather than allowing any free-form JSON schema, this RFC defines a set of JSON s
 
 # Motivation
 
 [motivation]: #motivation
 
-The Block Protocol operates in-between parties that otherwise have limited to no communication. A block developer may not even be aware of the existence of an embedding application or the data within it, and in some scenarios the developers and owners of an embedding application may not be aware of the existence of a block (in the case of dynamic selection and loading at run-time). This is a common occurrence already in technology, whether it be library authors not knowing what projects are built with their work, or developers using libraries without utilizing pathways for communication (online forums or otherwise). This is made possible through encoded contracts: function signatures, documentation, REST APIs, etc. These tools are ways of asynchronously, and often autonomously, declaring one side of the communication channel. In such a system of such imperfect information, the standardization of communication mediums is essential to effective function.
+The Block Protocol operates in-between parties that otherwise have limited to no communication. A block developer may not even be aware of the existence of an embedding application or the data within it, and in some scenarios the developers and owners of an embedding application may not be aware of the existence of a block (in the case of dynamic selection and loading at run-time). This is a common occurrence already in technology, whether it be library authors not knowing what projects are built with their work, or developers using libraries without utilizing pathways for communication (online forums or otherwise). This is made possible through encoded contracts: function signatures, documentation, REST APIs, etc. These tools are ways of asynchronously, and often autonomously, declaring one side of the communication channel. In a system of such imperfect information, the standardization of communication mediums is essential to effective function.
 
 By getting both parties to agree to a prescriptive standard, new technological possibilities become viable. The Block Protocol is an example of this, enabling a new class of applications, ones that are able to dynamically load complex front-end components at run-time which are immediately usable and functional, even if the component was previously unknown to the application.
 
-The foundation of this capability is the standardization of the expression of requirements. Most notably, the block and entity schemas expressed through the graph service allow for dynamic resolution of expected input data.
+The foundation of this capability is standardizing how requirements are expressed. Most notably, the block and entity schemas expressed through the graph service allow for dynamic resolution of expected input data.
```

Comment:
I think we should avoid changes of grammar like this because it feels like wasted time when it doesn't affect the content of the document. However, because it's been made, I don't think this phrasing is as grammatically correct. It sounds like the foundation is the thing that standardizes, rather than the standardization of X _is_ the foundation.

Reply:
@benwerner01 in case this has gotten lost